postgres/database.go

```diff
@@ -95,18 +138,57 @@ func (db *DB) Migrate(folder embed.FS) error {
 	return nil
 }
 
-// Migrate the database down a single migration using the specified embedded migration folder.
-func (db *DB) MigrateDown(folder embed.FS) error {
-	goose.SetBaseFS(folder)
+// Migrate the database down a single step using the specified embedded migration folder.
+// "folder" specifies the location of the folder containing sql files within the embed.FS
+// To only run the Apollo migrations, set migrations to nil
+func (db *DB) MigrateDown(migrations *embed.FS, folder string) error {
```

Comment:
Bijna heel deze functie is hetzelfde als de up migration (tot dat je effectief het `Down` commando uitvoert), mss kan je het gezamelijke deel extraheren naar een aparte functie, die dan de provider als return value heeft?

Reply:
Ah, inderdaad. Ik heb het er uitgetrokken, merci!